dandi/cli/cmd_download.py

```diff
@@ -48,12 +55,16 @@
 )
 @click.argument("url", nargs=-1)
 @map_to_click_exceptions
-def download(url, output_dir, existing, jobs=6, develop_debug=False):
+def download(url, output_dir, existing, jobs=6, format="pytout"):
```

Comment:
I don't think there's a situation where click would leave `format` unfilled, and if there is, shouldn't the default be "pyout", not "pytout"?

Reply:
oh... so we can avoid duplicating default values within `kwargs`?! let met just strip then them away from `download` and we could do to the rest in a separate PR.  THANK YOU! done in 4798e49fe49be2ea54599bd02d0009b1716e5044